cmd/dra-example-kubeletplugin/state.go

```diff
@@ -101,6 +101,8 @@ func (s *DeviceState) Prepare(claimUID string, allocation nascrd.AllocatedDevice
 	switch allocation.Type() {
 	case nascrd.GpuDeviceType:
 		prepared.Gpu, err = s.prepareGpus(claimUID, allocation.Gpu)
+	default:
+		err = fmt.Errorf("wrong allocated device type: %v", allocation.Type())
```

Comment:
Can we make this the following to be consistent with line 115:
```
		return fmt.Errorf("unknown device type: %v", devices.Type())
```

Reply:
Done.